lib/chef/chef_fs/file_system/chef_server/nodes_dir.rb

```diff
@@ -28,9 +28,9 @@ module ChefServer
         class NodesDir < RestListDir
           # Identical to RestListDir.children, except supports environments
           def children
-            @children ||= root.get_json(env_api_path).keys.sort.map do |key|
+            @children ||= root.get_json(env_api_path).keys.sort.map { |key|
               make_child_entry(key, true)
-            end
+            }
```

Comment:
I'm fairly sure mike_child_entry actually does a side-effect so this looks incorrectly applied again.